src/sagemaker/serve/app.py

```diff
@@ -0,0 +1,80 @@
+"""FastAPI requests"""
+
+from __future__ import absolute_import
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+
+try:
+    import uvicorn
+
+except ImportError:
+    logger.error("To enable in_process mode for Transformers install uvicorn from HuggingFace hub")
+
+
+try:
+    from transformers import pipeline
+
+    generator = pipeline("text-generation", model="gpt2")
+
+except ImportError:
+    logger.error(
+        "To enable in_process mode for Transformers install transformers from HuggingFace hub"
+    )
+
+
+try:
+    from fastapi import FastAPI, Request
+
+    app = FastAPI(
+        title="Transformers In Process Server",
+        version="1.0",
+        description="A simple server",
+    )
+
+    @app.get("/")
+    def read_root():
+        """Placeholder docstring"""
+        return {"Hello": "World"}
+
+    @app.get("/generate")
+    async def generate_text(prompt: Request):
+        """Placeholder docstring"""
+        logger.info("Generating Text....")
```

Comment:
nit: to keep user experience clean. Let's reduce logging. Let's remove these two logs in `async def generate_text(prompt: Request):` method.

Reply:
Thank you, I will remove.